### ISSUE TITLE
`write` api takes a lot of time after completion

### ISSUE THREAD

**Issue by NivedhaSenthil** [getgauge/taiko#277]
```
(async () => {
    try {
        await openBrowser();
        await goto("google.com");
        await write("hi");
    } catch (e) {
        console.error(e);
    } finally {
        await closeBrowser();
    }
})();
```
write in the above script takes more than 30secs after the text is actually written.

**Comment by klaussilveira** [comment]
You might want to disable `waitForNavigation`, or tweak the timeouts: https://taiko.gauge.org/#write

**Comment by NivedhaSenthil** [comment]
Wait for navigation is optimised now. The unresolved promises were handled properly. Fix available in d29e749.  Some more flow is tracked as part of https://github.com/getgauge/taiko/issues/304.
